clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/DockerRegistryConfiguration.groovy

```diff
@@ -48,4 +51,9 @@ class DockerRegistryConfiguration {
   DockerRegistryHealthIndicator dockerRegistryHealthIndicator() {
     new DockerRegistryHealthIndicator()
   }
+
+  @Bean
+  String dockerApplicationName(@Value('${Implementation-Version:Unknown}') String implementationVersion) {
```

Comment:
Last time I tried, I couldn't get the core module to load this value. But I'll give it another shot.